src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -293,11 +293,10 @@ public void Track(ITelemetry telemetry)
             if (this.IsEnabled())
             {
                 this.Initialize(telemetry);
-                
-                this.WriteTelemetryToDebugOutput(telemetry);
```

Comment:
It is an interesting discussion. How to distinguish between important telemetry that was aggregated and sampled and non-important telemetry or trash that user has chosen to filter away. If let's say I filtered out Verbose traces for non-failed requests. Does it mean Visual Studio should not display it? Does it mean customer do not care about this telemetry?

Also if we will decide that there is a way to distinguish bad telemetry - we need to apply the same logic of not writing it to this call: ```RichPayloadEventSource.Log.Process(telemetry);```